lib/bambora/bank/batch_report_resource.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module Bambora
+  module Bank
+    ##
+    # For making requests to the /scripts/reporting/report.aspx endpoint
+    #
+    # @see https://dev.na.bambora.com/docs/guides/batch_payment/report/
+    class BatchReportResource
+      DEFAULT_REQUEST_PARAMS = {
+        rpt_format: 'JSON',
```

Comment:
Does this report always return JSON, even if you use the XML client to retrieve it? Is this to separate the return type from the client?

Reply:
I discovered that it will return XML if you encounter an error. But a success response will always return JSON. That is correct, that this is meant to separate the return type from the client. 

The client inspects the response's `Content-Type` to decide how to parse the response.

I didn't feel it was necessary to add an XML parser to handle the case where we get an error response back since we raise an error and print the response body in the message.